roachtest: sqlsmith/setup=seed/setting=no-mutations failed

[(roachtest).sqlsmith/setup=seed/setting=no-mutations failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2457321&tab=buildLog) on [master@8ff64f5d897a9c7b39ca437193a11a7300cecbc1](https://github.com/cockroachdb/cockroach/commits/8ff64f5d897a9c7b39ca437193a11a7300cecbc1):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/sqlsmith/setup=seed/setting=no-mutations/run_1
	sqlsmith.go:181,sqlsmith.go:211,test_runner.go:755: error: pq: internal error: crdb_internal.unsafe_delete_descriptor(): log-event: desc 12: parentID 1 does not exist
		stmt:
		SELECT
			CASE WHEN true THEN tab_9153.col_22243 ELSE tab_9153.col_22243 END AS col_22244
		FROM
			(
				WITH
					with_3702 (col_22240)
						AS (SELECT * FROM (VALUES (e'YJ\x15cRnhc=':::STRING), ('3':::STRING)) AS tab_9150 (col_22240)),
					with_3703 (col_22241)
						AS (
							SELECT
								*
							FROM
								(VALUES (665109327:::OID), (57723738:::OID), (3115403075:::OID)) AS tab_9151 (col_22241)
						)
				SELECT
					count(*) AS col_22242
				FROM
					defaultdb.public.seed@[0] AS tab_9152
				WHERE
					true
				GROUP BY
					tab_9152._int8, tab_9152._float8, tab_9152._date
				HAVING
					crdb_internal.unsafe_delete_descriptor(tab_9152._int8::INT8)::BOOL
				ORDER BY
					tab_9152._int8, tab_9152._float8 ASC
				LIMIT
					74:::INT8
			)
				AS tab_9153 (col_22243)
		ORDER BY
			tab_9153.col_22243 DESC, tab_9153.col_22243;

```

<details><summary>More</summary><p>


Artifacts: [/sqlsmith/setup=seed/setting=no-mutations](https://teamcity.cockroachdb.com/viewLog.html?buildId=2457321&tab=artifacts#/sqlsmith/setup=seed/setting=no-mutations)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Asqlsmith%2Fsetup%3Dseed%2Fsetting%3Dno-mutations.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/internal/sqlsmith/schema.go
+++ b/pkg/internal/sqlsmith/schema.go
@@ -417,7 +417,8 @@ var functions = func() map[tree.FunctionClass]map[oid.Oid][]function {
 		case "pg_sleep":
 			continue
 		}
-		if strings.Contains(def.Name, "crdb_internal.force_") {
+		if strings.Contains(def.Name, "crdb_internal.force_") ||
+			strings.Contains(def.Name, "crdb_internal.unsafe_") {
 			continue
 		}
 		if _, ok := m[def.Class]; !ok {

